pkg/webhooks/operator_webhooks.go

```diff
@@ -105,17 +106,19 @@ func (webhookConfig *IntegreatlyWebhookConfig) SetupServer(mgr manager.Manager)
 		return err
 	}
 
-	webhookServer := mgr.GetWebhookServer()
-	webhookServer.Port = webhookConfig.Port
-	webhookServer.CertDir = webhookConfig.CertDir
-
-	webhookConfig.scheme = mgr.GetScheme()
+	// Create a webhook server.
+	webhookServer := webhook.NewServer(webhook.Options{
```

Comment:
is mgr.Get same as mgr.New?

Reply:
I thought that here is the place where webhooks are created.
The reason of the changes  - that we can't now set directly Options (Port and CertDir) like it was before. It need to be done or in webhookServer creation time, or at Start time. So this code is creation webhookServer with required options.